clusterloader2/pkg/util/cluster.go

```diff
@@ -28,6 +28,18 @@ import (
 
 const keyMasterNodeLabel = "node-role.kubernetes.io/master"
 
+// Based on https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions.
```

Comment:
we could also mention https://kubernetes.io/docs/reference/labels-annotations-taints/ here

Reply:
missing taints:
- `node.kubernetes.io/pid-pressure`
- `node.cloudprovider.kubernetes.io/shutdown`

Reply:
Thanks for pointing this out.

Added the aforementioned taints and referenced the doc in the comment.